docs-site/custom.css

```
@@ -0,0 +1,134 @@
+  text-decoration: none;
+}
+
+.footer__copyright {
+  border-top: 1px solid var(--ifm-toc-border-color);
+  color: var(--ifm-color-content-secondary);
+  margin-top: 1rem;
+  padding-top: 1rem;
+}
+
+/* Force override Infima defaults (which use high-specificity selectors). */
+:root:not(#\#):not(#\#):not(#\#) {
```

Stylelint (17.3.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)